### PR TITLE
ENH: updates to editor & reviewer onboarding pages

### PR DESCRIPTION
**Pull request by lwasser**
i added commits on top of #290 but something is super funky with those commits! it should be ok however once we merge. 

what is actually missing from this page now is reviewer onboarding!! so i'll open this in draft.

### CODE REVIEW

File: how-to/onboarding-guide.md [hunk 1]
```diff
@@ -122,81 +204,62 @@ Best,
...
   contributor_type:
-    - current editor # Make sure emeritus editor is here
-    - contributor
-  packages-submitted: [""]
-  packages-reviewed: ["pystiche"]
-  packages-editor: [""]
+    - editor # This field will be automatically updated after their first review
+    - guidebook-contrib
+    - reviewer
 ```
```
Comment (lwasser) [inline how-to/onboarding-guide.md:265]: TODO: add - onboarding new reviewers section!

**Comment by lwasser** [comment]
ok @kierisi please give this a read - i've threaded in the community manager help pieces in both the reviewer and editorial pages and cleaned up the commits a bit.  

see if the process we have in place for editors now makes sense where they look using their own networks and our community and then go to the community manager if they are stuck. the same would go for editors seeking reviewers.

File: how-to/finding-reviewers.md [hunk 2]
```diff
@@ -1,53 +1,99 @@
...
+  volunteered to review for pyOpenSci. The link for this document is pinned to
+  the pyOpenSci Slack `#private-editorial-channel`.
 * Ping other editors for ideas.
-* Look for users of the package or the data source/upstream service the package connects to (via their opening issues in the repository, starring it, citing it in papers, talking about it on Twitter).
-* You can also search for authors/maintainers of related packages on [PyPI](https://pypi.org/search/).
-* Post on Twitter and ensure pyOpenSci retweets your post.
+* Post in the pyOpenSci Slack `#software-review` channel.
+* Look for users of the package or the data source/upstream service the
+  package connects to (via their opening issues in the repository, starring
+  it, citing it in papers, talking about it on Twitter).
```
Comment (kierisi) [inline how-to/finding-reviewers.md:25]:
```suggestion
  it, citing it in papers, talking about it on social media).
```

File: how-to/finding-reviewers.md [hunk 3]
```diff
@@ -1,53 +1,99 @@
...
 
 (reviewer-diversity)=
-### Reviewer diversity should be prioritized
-Try to balance your sense of the potential reviewer’s experience against the complexity of the package.
+### Reviewer Diversity Should Be Prioritized
+
+Try to balance your sense of the potential reviewer’s experience against the
+complexity of the package.
 
 * **Diversity** - if you have two reviewers, both shouldn’t be cis white males.
```
Comment (kierisi) [inline how-to/finding-reviewers.md:54]:
```suggestion
* **Diversity** - if you have two reviewers, only one should be a cis white male.
```

Comment (kierisi) [reply how-to/finding-reviewers.md:54]: bit of a nit, but I'm trying to move my language away from "negatives" and more towards positives. 

related: are we not considering LGBTQIA+ identity as part of diversity?

File: how-to/finding-reviewers.md [hunk 4]
```diff
@@ -1,53 +1,99 @@
...
+review experience.
+
+It is useful for reviewers to not only review the technical content of a
+package, but also to review the documentation and package installation process
+for usability.
+
+If a new reviewer is interested in becoming a reviewer but would like some
+support, do the following:
+
+1. Reach out to the **editor in chief** & **peer review lead** to let them
```
Comment (kierisi) [inline how-to/finding-reviewers.md:87]:
```suggestion
1. Reach out to the **Editor in Chief** & **Peer Review Lead** to let them
```

File: how-to/finding-reviewers.md [hunk 5]
```diff
@@ -1,53 +1,99 @@
...
+   know that you have a reviewer that would like to work with a mentor.
+2. They will then post in the `#software-review` channel to see if anyone is
+   available to provide support to the new reviewer.
+3. If you don't get any responses from the post above, reach out to our
+   community manager who will try to find someone to fill the mentorship role!
```
Comment (kierisi) [inline how-to/finding-reviewers.md:92]:
```suggestion
   Community Manager who will try to find someone to fill the mentorship role!
```

File: how-to/onboarding-guide.md [hunk 6]
```diff
@@ -1,81 +1,196 @@
...
+### Types of Editors
+
+#### New Editors Start as "Guests"
+
+A new editor will be considered a guest editor for the first 3 months of their
+tenure and/or until they have completed their first review. Once they have
+completed a review, they can be considered a full editor as deemed appropriate
+by the software review lead and the current editorial board.
+
+#### Adhoc Guest Editors
```
Comment (kierisi) [inline how-to/onboarding-guide.md:35]:
```suggestion
#### _ad hoc_ Guest Editors
```

File: how-to/onboarding-guide.md [hunk 7]
```diff
@@ -1,81 +1,196 @@
...
+
+Adhoc editors are editors with specific skill sets that are brought in to lead
```
Comment (kierisi) [inline how-to/onboarding-guide.md:37]:
```suggestion
_ad hoc_ editors are editors with specific skill sets that are brought in to lead
```

File: how-to/onboarding-guide.md [hunk 8]
```diff
@@ -1,81 +1,196 @@
...
+a single review. Examples of when there might be a need for an ad-hoc editor
```
Comment (kierisi) [inline how-to/onboarding-guide.md:38]:
```suggestion
a single review. Examples of when there might be a need for an _ad hoc_ editor
```

File: how-to/onboarding-guide.md [hunk 9]
```diff
@@ -1,81 +1,196 @@
...
+If a potential volunteer does not have prior software editorial experience, we
+offer a **mentorship process**. Editor mentorship is where someone with
+existing editorial experience mentors the new editor through their first
+review(s).
+
+### Recruiting New Editors
+
+Recruiting new editors and maintaining a sufficient and well-balanced
+editorial board is the responsibility of the
+[Software Review Lead](https://www.pyopensci.org/handbook/governance/structure.html#software-review-lead)
```
Comment (kierisi) [inline how-to/onboarding-guide.md:66]: flagging that it was earlier referred to as peer review lead - I'm assuming these are the same position?

File: how-to/onboarding-guide.md [hunk 10]
```diff
@@ -1,81 +1,196 @@
...
+    review to pyOpenSci
+  * Reviewers who have already led reviews for pyOpenSci
+  * Past guest editors
+* Colleagues that you know who have reviewed for the Journal of Open Source
+  Software (JOSS) or rOpenSci who have Python expertise
+* Community members in the pyOpenSci Slack (please post a call in our
+  `#software-review` channel)
+
+When all of the above fails to return a good new editor candidate, you can find
+support from our pyOpenSci community manager who will post an open call on our
```
Comment (kierisi) [inline how-to/onboarding-guide.md:98]:
```suggestion
support from our pyOpenSci Community Manager who will post an open call on our
```

Comment (kierisi) [reply how-to/onboarding-guide.md:98]: while it's _technically_ correct to keep job titles all lower case, it's culturally//stylistically preferred to have them capitalized

File: how-to/onboarding-guide.md [hunk 11]
```diff
@@ -1,81 +1,196 @@
...
+
+  * That you’re looking for editors.
+  * Any specific domains or technical skills you need an editor to have, or if it’s a general call for editors.
+  * Any specific packages that need an editor (include links to a package submission that they would lead if possible).
+
+:::{tip}
+It's helpful to post in our `#software-review` channel rather than
+an individual Direct Message (DM) because:
+
+* It allows other editors to see the type of expertise that you are requesting and that we offer this support from our community manager.
```
Comment (kierisi) [inline how-to/onboarding-guide.md:154]:
```suggestion
* It allows other editors to see the type of expertise that you are requesting and that we offer this support from our Community Manager.
```

File: how-to/onboarding-guide.md [hunk 12]
```diff
@@ -1,81 +1,196 @@
...
+* It allows other community members who may know of potential editors to see the request and potentially respond.
+:::
+
+Our Community Manager interacts with the broader community and partner communities and as such often interacts with people who may be interested in joining our pyOpenSci review team. The community manager may from time to time meet community members who might make great editors or reviewers. In those cases they will share that information with the current EiC and peer review lead.
```
Comment (kierisi) [inline how-to/onboarding-guide.md:158]:
```suggestion
Our Community Manager interacts with the broader community and partner communities and as such often interacts with people who may be interested in joining our pyOpenSci review team. The Community Manager may from time to time meet community members who might make great editors or reviewers. In those cases they will share that information with the current Editor in Chief and Peer Review Lead.
```

File: how-to/onboarding-guide.md [hunk 13]
```diff
@@ -1,81 +1,196 @@
...
+:::
 
-In this case, you may consider using our internal reviewer signup list to see
-if someone who signed up to be a reviewer might want to serve as an editor.
 
 ## Process for inviting a new editor
 
 * Editorial board candidates most often start as guest editors.
-* After 3 months or their first review (which ever comes first) assess how the
+* After 3 months or their first review (which ever comes first), the peer review lead working with the EiC will assess how the
```
Comment (kierisi) [inline how-to/onboarding-guide.md:193]:
```suggestion
* After 3 months or their first review (which ever comes first), the Peer Review Lead working with the EiC will assess how the
```

File: how-to/onboarding-guide.md [hunk 14]
```diff
@@ -122,81 +237,62 @@ Best,
...
-    * [dev_guide authors list](https://github.com/ropensci/dev_guide/blob/main/index.Rmd)
-    * [dev_guide chapter introducing software review](https://github.com/ropensci/dev_guide/blob/main/softwarereview_intro.Rmd) (at two locations in this file, as editors and a bit below to remove them from the reviewers list)
-    * [software-review README](https://github.com/ropensci/software-review/blob/main/README.Rmd) (in two places in this file as well)
-Both the dev_guide and software-review README are automatically knit via continuous integration. -->
+:::{note}
+We have a bi-weekly cron job that parses through existing issues and grabs the names of editors, reviewers and authors. However, if you want the editors name to be listed on the website prior to a review beginning and/or sooner than the cron job might pick up their name, then we suggest that you add their name, and GitHub username and title to the contributors.yml file through a pull request.
 
-<!-- * Add editors to https://github.com/orgs/ropensci/teams/editors/members -->
+You do not need to fill out all of the elements of the YAML file - only the name, github user field and the `editorial_board: true` key:value pair.
```
Comment (kierisi) [inline how-to/onboarding-guide.md:258]:
```suggestion
You do not need to fill out all of the elements of the YAML file - only the name, GitHub user field and the `editorial_board: true` key:value pair.
```

**Review by kierisi — COMMENTED** [review]
looks good! made some grammatical suggestions.

File: how-to/finding-reviewers.md (hunk 3 above)
Comment (lwasser) [reply how-to/finding-reviewers.md:54]: we are definitely not excluding groups in our diversity metrics.

File: how-to/onboarding-guide.md (hunk 9 above)
Comment (lwasser) [reply how-to/onboarding-guide.md:66]: it is. i'll make it consistent locally!

**Pull request merged**